test_conformance/subgroups/subgroup_common_templates.h

```diff
@@ -560,26 +568,30 @@ template <typename Ty, ShuffleOp operation> struct SHF
             y += nw;
             m += 4 * nw;
         }
-        log_info("  sub_group_%s(%s)... passed\n", operation_names(operation),
-                 TypeManager<Ty>::name());
         return TEST_PASS;
     }
 };
 
 template <typename Ty, ArithmeticOp operation> struct SCEX_NU
 {
+    static void log_test(const WorkGroupParams &test_params,
+                         const char *extra_text)
+    {
+        std::string func_name;
+        test_params.work_items_mask.any()
```

Comment:
first run `log_test` do not have `work_items_mask` loaded and function name and `sub_group_scan_exclusive` is printed instead of `sub_group_non_uniform_scan_exclusive`

Reply:
Thanks!  I have fixed this to test divergence_mask_arg instead.